.builds/openbsd.yml

```diff
@@ -23,8 +23,9 @@ environment:
 
 tasks:
 - build-deps: |
-    export AUTOCONF_VERSION=2.69
-    export AUTOMAKE_VERSION=1.15
+    export AUTOCONF_VERSION=2.71
+    export AUTOMAKE_VERSION=1.16
+    export VERBOSE=1
```

Comment:
@glacambre Actually should verbose not be here?

Reply:
Yup, I think we can remove it, I only added it out of despair :).